src/app/core/signing.guard.ts

```diff
@@ -29,55 +30,59 @@ export class SigningGuard implements CanActivate {
     next: ActivatedRouteSnapshot,
     state: RouterStateSnapshot): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
 
-    // Regex Extraction of 'uid' and 'key' params
-    try {
-      const linkStr = window.location.href;
-      SigningGuard.uid = linkStr.match(/uid=([^&]*)/)[1];
-      SigningGuard.key = linkStr.match(/key=([^&]*)/)[1];
-    } catch (error) {
-      // Displaying alert is not necessary since an error is only thrown when one of the query params is missing/invalid = invalid link
-      this.router.navigate([ '/invalid-link' ]).catch(error => AuthService.displayError(error));
-    }
+    if (this.maintenance) {
+      this.router.navigate([ '/maintenance' ]);
```

Comment:
Missing return statement
```suggestion
      this.router.navigate([ '/maintenance' ]);
      return false;
```